doc/users/prev_whats_new/whats_new_3.5.0.rst

```diff
@@ -14,6 +14,26 @@ the :ref:`github-stats`.
 Figure and Axes creation / management
 =====================================
 
+``subplot_mosaic`` supports simple Axes sharing
+-----------------------------------------------
+
+`.Figure.subplot_mosaic`, `.pyplot.subplot_mosaic` support *simple* Axes
+sharing (i.e., only `True`/`False` may be passed to *sharex*/*sharey*). When
+`True`, tick label visibility and Axis units will be shared.
+
+.. plot::
+    :include-source:
+
+    layout = [
```

Comment:
Change to `mosaic` as we've renamed the parameter to distinguish from constrained/tight layout?